src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,76 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static Random random = new Random();
+    private static ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
These variables should not be static
```suggestion
    private Random random = new Random();
    private ColorSupplier colorSupplier = new ColorSupplier();
```